packages/react-ui-core/src/Button/ToggleButton.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
```

Comment:
Is this branch out of date? It looks like we already have a `ToggleButton.js` file in master.